OneCommunityGlobal/HighestGoodNetworkApp
# Faye Deleted src/components/ProfileLinks/** according to PR1671

# Description
Updated lint ignore file and Deleted profile link file according to PR1671.
https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/1671

## How to test:
check into current branch
do npm install and ... to run this PR locally
Clear site data/cache
log as admin user
check all the function is normal

## Screenshots or videos of changes:

https://github.com/user-attachments/assets/ea3ff8b5-51e9-4175-8b58-2624762e63be


## Note:
Include the information the reviewers need to know.
